app/policies/division_policy.rb

```diff
@@ -18,4 +18,30 @@ def update?
   def destroy?
     division_admin(division: @record)
   end
+
+  def select?
+    show?
+  end
+
+  class Scope < Scope
```

Comment:
ApplicationPolicy.show? verifies that the scope can resolve the given record id.
And I've added a 'Scope' block to the division_policy_spec, and I've factored out a division_owned_scope set of shared tests.  
I'm guessing you may have some feedback on how I could better factor the rspec code, so I'll give you a chance to review before I merge down to develop.